src/main/java/hudson/plugins/sloccount/model/cloc/ClocTotal.java

```diff
@@ -23,6 +23,9 @@
 
     /**
      * Constructor.
+     * @param blank blank
+     * @param comment comment
+     * @param code code
```

Comment:
> But older environment for build did not check JavaDoc results. New one complains for it - I did not find out all meaning for all parameters.

It prints warnings, but it does not block the build/release. So it is probably fine to skip them. It is also fine to leave the current change IMHO.